internal/network/command_dns_record_list.go

```diff
@@ -0,0 +1,99 @@
+package network
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/confluentinc/cli/v3/pkg/ccloudv2"
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/errors"
+	"github.com/confluentinc/cli/v3/pkg/examples"
+	"github.com/confluentinc/cli/v3/pkg/output"
+)
+
+func (c *command) newDnsRecordListCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "list",
+		Short: "List DNS records.",
+		Args:  cobra.NoArgs,
+		RunE:  c.dnsRecordList,
+		Example: examples.BuildExampleString(
+			examples.Example{
+				Text: `List DNS records with display names "my-dns-record-1" or "my-dns-record-2.`,
+				Code: "confluent network dns record list --names my-dns-record-1,my-dns-record-2",
+			},
+		),
+	}
+
+	cmd.Flags().String("gateway", "", "Gateway ID.")
```

Comment:
I think we are changing this to a `MultipleSearchFilter`, will have to update the api spec and sdk

Reply:
I will check with the team and get back to you

Reply:
updated the sdk - https://github.com/confluentinc/ccloud-sdk-go-v2-internal/releases/tag/networking-access-point%2Fv0.0.2

Reply:
Switched to new SDK.